### PR TITLE
add gnex into orgs

### PR DESCRIPTION
**Pull request by alphaKAI**
I added GNEX Ltd. into organizations

### CODE REVIEW

**Comment by ZombineDev** [comment]
Since list is sorted alphabetically, can you move GNEX between Funkwerk and Infognition? Otherwise, the change looks good to me.

File: orgs-using-d.dd [hunk 1]
```diff
@@ -159,6 +159,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG GNEX, https://globalnet-ex.com, gnex.png,
+        Digital Marketing Solutions Company.,
+        $(LINK_ROW
+            $(FA_GITHUB gnexltd) $(FA_SEPARATOR)            
+            $(FA_HIRING https://globalnet-ex.com/employment/engineer)
```
Comment (ZombineDev) [inline orgs-using-d.dd]: You need to remove `https://`, otherwise, the link doesn't work, because of how ddoc handles `:`.

**Comment by alphaKAI** [comment]
Thx, i've fixed order of list, and removed `https://`, then i re-pushed

**Comment by alphaKAI** [comment]
@ZombineDev Cool! Can you review to merge this PR?
GitHub says merging is blocked because there is no approved review.

**Comment by ZombineDev** [comment]
Unfortunately, I can't merge this, because I don't have rights.
Ping @CyberShadow @MartinNowak @JackStouffer.

**Comment by ZombineDev** [comment]
BTW, can you squash your two commits into one? Individual commits are good for review, but when the code is ready to be merged it is better to squash fix-up commits that don't have add value to the history of the repository. 
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by alphaKAI** [comment]
@ZombineDev I integrated two commits into one with given URL by you.

**Comment by ZombineDev** [comment]
Thanks, saying from experience, your PR should be merged in a couple of days max.

**Pull request merged**